core/string/shared/each_line.rb

```diff
@@ -133,4 +133,14 @@
   it "raises a TypeError when the separator is a symbol" do
     lambda { "hello world".send(@method, :o).to_a }.should raise_error(TypeError)
   end
+
+  ruby_version_is '2.4' do
+    context "when `chomp` keyword argument is passed" do
+      it "removes carriage return characters" do
```

Comment:
In the example below, line feed (\n) is used, not carriage return (\r).
But it would be interesting to see if it removes `\r\n` too. Could you add an example for that?
One way to summarize both is to say it "removes new line characters".

Reply:
@eregon done